---
layout: software
title: Software
icon: fa-pencil-alt
order: 4
---


<!-- metaGOflow -->

<p align="center">
   <a href="https://metagoflow.readthedocs.io">
      <img src="https://raw.githubusercontent.com/hariszaf/metaGOflow-use-case/gh-pages/assets/img/metaGOflow_logo_italics.png"  width="300" align="left" >
   </a>
</p>
<br/>
## metaGOflow: one shotgun at a time
*Main page on* [*metagoflow.readthedocs.io*](https://metagoflow.readthedocs.io).
*metaGOflow is reposited on [GitHub](https://github.com/emo-bon/MetaGOflow) as well as on [WorkflowHub](https://workflowhub.eu/workflows/384).

[metaGOflow](https://hariszaf.github.io/pema_documentation/) is a HPC-oriented, containerized based workflow for the analysis of shotgun metagenomes.
It serves as a flexible and easy-to-use workflow that can be broadly used for one-sample-at-a-time analysis of shotgun metagenomics data.


<!-- PEMA -->

<p align="center">
   <a href="http://pema.hcmr.gr/">
      <img src="https://i.paste.pics/870189fadf668a958c8aac83f38e799c.png"  width="300" align="left" >
   </a>
</p>
<br/>
## PEMA: a flexible Pipeline for Environmental DNA Metabarcoding Analysis of the 16S/18S rRNA, ITS and COI marker genes
*Main page on* [*pema.hcmr.gr*](http://pema.hcmr.gr/).
*PEMA is reposited in* [*Docker Hub*](https://hub.docker.com/r/hariszaf/pema) *as well as in* [*Singularity Hub*](https://singularity-hub.org/collections/2295).

[Pipeline for Environmental Metabarcoding Analysis (PEMA)](https://hariszaf.github.io/pema_documentation/) is a HPC-centered, containerized assembly of key metabarcoding analysis tools. PEMA combines state-of-the art technologies and algorithms with an easy to get-set-use framework, allowing researchers to tune thoroughly each study thanks to roll-back checkpoints and on-demand partial pipeline execution features


<!-- RPEGO -->


<p align="center">
   <a href="http://prego.hcmr.gr/">
     <img src="https://prego.hcmr.gr/icons/prego.png"  width="350" align="left"  Hspace="30">
   </a>
</p>

## Mining microbiomes!
<br/>
Process, environment, organism ([PREGO](http://prego.hcmr.gr/)) is a systems-biology approach to elucidate ecosystem function at the microbial dimension. To understand how ecosystems function it is fundamental to understand what processes occur in which environments (where), and which organisms carry them out (who). Its main motivation is to provide a one-stop-shop for researchers interested in searching and visually exploring such what - where - who associations.

<!-- VOLESTI -->

<p align="bottom">
   <a href="https://github.com/GeomScale/volume_approximation">
     <img src="https://i.ibb.co/kqT6rP5/imageedit-70-4833876726.png"  width="300" align="left" >
   </a>
</p>
<br/>

## a C++ library for high-dimensional sampling and volume approximation for modelling metabolic networks.

<br/>

VolEsti is a C++ library for volume approximation and sampling of convex bodies (e.g. polytopes) with an R and limited python interface. VolEsti is part of the [GeomScale](https://geomscale.github.io/) project. <br/>
Along with the GeomScale group, we are building a complete toolobx  for the analysis of metabolic network.
Our aim is  to address the challenges of high dimensional networks (e.g the human Recon 3D) as well as those rising from networks of networks, just like in the case of metabolic networks of microbial communities.



<!-- DARN -->
<p align="bottom">
   <a href="https://github.com/hariszaf/darn">
      <br/>
      <img src="https://raw.githubusercontent.com/hariszaf/darn/main/figures/darn_logo.png"  width="300" align="left" >
   </a>
</p>
## Dark mAtteR iNvestigator: diving into the dark matter of your COI sequences
<br/><br/>
The [Dark mAtteR iNvestigator tool (DARN)](https://github.com/hariszaf/darn/) uses a COI reference tree of life we built, to assign your sequences to the 3 domains of life.
Its purpose is not to provide you with certain taxonomic assignment but to give an overview of the species present.