---
layout: post
title: "A Pipeline for Environmental DNA Metabarcoding Analysis"
subtitle: "gitlab to github repo clone with commit logs"
category: software
tags: amplicon workflow
image: /assets/img/pema/logo.png
githubButton: https://github.com/hariszaf/pema
documentation: https://github.com/hariszaf/pema_documentation
---

PEMA is a BigDataScript-based workflwow for the analysis of amplicon data. 

PEMA supports the metabarcoding analysis of four marker genes, 16S rRNA (Bacteria), ITS (Fungi) as well as COI and 18S rRNA (metazoa). 
As input, PEMA accepts .fastq.gz files as returned by Illumina sequencing platforms.
Since the `v.2.1.4` release, PEMA supports also the analysis of the 12S rRNA marker gene!

PEMA is also the main workflow performed in the framework of the [Autonomous Reef Monitoring Structures (ARMS) programme: long-term monitoring of invasive marine species](https://www.lifewatch.eu/internal-joint-initiative/validation-cases/long-term-monitoring-of-invasive-marine-species/) an Internal Joint Initiative of [LifeWatch ERIC](https://www.lifewatch.eu). 


PEMA always needs your help to keep up-to-date and integrate new features. 
If you are interested in contributing to the project, feel free to open an Issue, or a PR on GitHub or contact me for more! :computer:


Read more on [PEMA publication](/publications/pema.html){:.heading.flip-title}
{:.read-more}
Read also a review on [relative amplicon workflows](/publications/pile.html){:.heading.flip-title}
{:.read-more}

