---
layout: post
title: Arsenal
category: resources
image: https://media.licdn.com/dms/image/v2/D561BAQGxm3Cp1Lc-tQ/company-background_10000/company-background_10000/0/1654722729907/lyda_hill_department_of_bioinformatics_cover?e=2147483647&v=beta&t=XF1Z5LhEVtcFFy_D-4E7gwz3qDv6xoG9qe0YVSdOXQI
tags: tools
---

This is my favorite resources' page
<!-- To make life easier, we use the Markdown All in One vs-code add on
You can make a shitty table first and then by Ctrl+Shift+I make it pretty 
Here you can find more: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one -->

<!-- Ctrl+shift+p and then run: `Create Table of Contents`. this also comes from the "all in one" extension -->
**TABLE OF CONTENTS**
- [Metabolic modeling](#metabolic-modeling)
- [Genomics and Metagenomics](#genomics-and-metagenomics)
- [Databases](#databases)
- [Statistics for microbiome analysis](#statistics-for-microbiome-analysis)
- [My dictionary](#my-dictionary)

---


## Metabolic modeling

You may have a look at this [*e-book*](https://hariszaf.github.io/notes-on-met-mod-analysis/tools.html) :notebook: I've been working on. 

## Genomics and Metagenomics

| Tool     | Description                                                                   | Architecture | Repo                                                          | DOI                                       |
| :------- | :---------------------------------------------------------------------------- | :----------- | :------------------------------------------------------------ | :---------------------------------------- |
| LocalHGT | ultrafast horizontal gene transfer detection from large microbial communities | stand-alone  | [GitHub](https://github.com/deepomicslab/LocalHGT#hgt-events) | [OA](https://doi.org/10.1093/nar/gkae515) |

<button class="copy-button">Copy Table</button>

## Databases

| Resource | Description | Link |  DOI |
| -------: | ----------: | ---: | ---: |
|     tree |         asd |  ads |  asd |

<button class="copy-button">Copy Table</button>

## Statistics for microbiome analysis
<!-- ✘ (&#10008) is a not available ; ✔ (&#10003) is a tick for yes -->
<!-- Template for copy-paste of new entry:

|  |  |  |  |  <a href="">✔</a> |   |
-->

| Tool            | Description                                                                                                                         | Architecture        | Repo                                                   |                        Documentation                         |                         DOI                          |
| :-------------- | :---------------------------------------------------------------------------------------------------------------------------------- | :------------------ | :----------------------------------------------------- | :----------------------------------------------------------: | :--------------------------------------------------: |
| MOFA2           | a factor analysis model that provides a general framework for the integration of multi-omic data sets in an unsupervised fashion.   | library (Python, R) | [ GitHub ](https://github.com/bioFAM/MOFA2)            |       <a href="https://biofam.github.io/MOFA2/">✔</a>        |   [OA](https://doi.org/10.1186/s13059-020-02015-1)   |
| coda4microbiome | analyzing microbiome data within the Compositional Data Analysis (CoDA) framework in both, cross-sectional and longitudinal studies | R package           | [GitHub](https://malucalle.github.io/coda4microbiome/) | <a href="https://malucalle.github.io/coda4microbiome/">✔</a> |   [OA](https://doi.org/10.1186/s12859-023-05205-3)   |
| multimedia      | multimodal mediation analysis of microbiome data                                                                                    | R package           | [GitHub](https://github.com/krisrs1128/multimedia)     |   <a href="https://krisrs1128.github.io/multimedia/">✔</a>   | [biorxiv](https://doi.org/10.1101/2024.03.27.587024) |


<button class="copy-button">Copy Table</button>



<!-- ## Mind-blowing papers  -->

## My dictionary 

In the following table I describe some terms I usually read/write about. 

| Term             | Description                                                                                                                                                                                                |
| :--------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| richness         | number of different taxa in a community                                                                                                                                                                    |
| evenness         | the commonness or rarity of a species                                                                                                                                                                      |
| mucin            | a family of high molecular weight, heavily glycosylated proteins (glycoconjugates) produced by epithelial tissues in most animals                                                                          |
| effective number | refers to the number of equally abundant species needed to obtain the same mean proportional species abundance as that observed in the dataset of interest (where all species may not be equally abundant) |
| copiotrophs      | taxa living in environments rich in nutrients                                                                                                                                                              |




<button class="copy-button">Copy Table</button>

