---
layout: post
title: "The Dark mAtteR iNvestigator (DARN) tool: getting to know the known unknowns in COI amplicon data"
subtitle: "Metabarcoding and Metagenomics"
category: publications
tags: amplicon
image:
  path: /assets/img/darn/darn-wf.png
related_posts:
  /_posts/software/2020-01-30-darn.md
paperButton: https://doi.org/10.3897/mbmg.5.69657
githubButton: https://github.com/hariszaf/darn/
documentation: https://hariszaf.github.io/darn/
---


The mitochondrial cytochrome C oxidase subunit I gene (COI) is commonly used in environmental DNA (eDNA) metabarcoding studies, especially for assessing metazoan diversity. 
Yet, a great number of COI operational taxonomic units (OTUs) or/and amplicon sequence variants (ASVs) retrieved from such studies do not get a taxonomic assignment with a reference sequence. 
To assess and investigate such sequences, we have developed the Dark mAtteR iNvestigator (DARN) software tool. 
For this purpose, a reference COI-oriented phylogenetic tree was built from 1,593 consensus sequences covering all the three domains of life. 
DARN makes use of this phylogenetic tree to investigate COI pre-processed sequences of amplicon samples to provide both a tabular and a graphical overview of their phylogenetic assignments. 

We demonstrate that a large proportion of non-target prokaryotic organisms, such as bacteria and archaea, are also amplified in eDNA samples and we suggest prokaryotic COI sequences to be included in the reference databases used for the taxonomy assignment to allow for further analyses of dark matter

Read more for the [`darn` software](/software/darn.html){:.heading.flip-title}
{:.read-more}

